.env.example

```
@@ -27,13 +27,19 @@ CLOUD_API_HOST=https://api.insforge.dev
 
 
 # Storage + Logs Configuration
-# Storage: Set AWS_S3_BUCKET, AWS_REGION + credentials to use S3 (otherwise uses local filesystem). Use AWS_ENDPOINT_URL for S3-compatible services.
-# Logs: Set AWS_REGION + credentials to enable CloudWatch logging (otherwise uses file-based logging)
+# Storage: Set AWS_S3_BUCKET, AWS_REGION + credentials to use S3 (otherwise uses local filesystem).
+# Logs: Set AWS_REGION + AWS credentials to enable CloudWatch logging (otherwise uses file-based logging)
 AWS_S3_BUCKET=
 AWS_REGION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_S3_BUCKET key

(UnorderedKey)